### ISSUE TITLE
Why is vibrato_rate not used?

### ISSUE THREAD

**Issue by bfs18** [magenta/midi-ddsp#38]
In my opinion, vibrato_rate (peak frequency) is more plausible than vibrato_extend (peak amplitude) to represent pitch pulsating. Why is vibrato_rate not used?

**Comment by lukewys** [comment]
Hi! In early experiments, we do tried using vibrato_rate as the control. However, we found that vibrato rate is not very effective for control, as the model barely learns to change its output vibrato frequency with different vibrato_rate inputs. It might be relevant to the noise and inaccuracy in vibrato rate detection, or it might be relevant to the RNN f0 generation model is having a hard time correlating vibrato_rate input to the output value, or it might be related to the in-continuous vibrato_rate value (because of discrete frequency resolution in DFT) and its distribution. Ethier way, we deprecated the vibrato_rate as control.
